src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,50 @@
+// components
+import { useState } from 'react';
+import { TodoForm } from '../TodoForm/TodoForm';
+import { TodoList } from '../TodoList/TodoList';
+
+// style
+import style from './Todo.module.scss';
+
+export function Todo() {
+  const [todos, setTodos] = useState([]);
+
+  const addTodo = (todo) => {
+    setTodos([todo, ...todos]);
+  };
+
+  const toggleTodo = (id) => {
+    setTodos(
+      todos.map((todo) => {
```

Comment:
it is not good idea to use map inside setter, to be better to implement this functional above